web/src/components/Layout.tsx

```diff
@@ -93,14 +95,13 @@ const Main = styled.main<{ fullWidth: boolean }>`
   }
 `
 
-const Aside = styled.aside<{ languageSelectorHeight: number }>`
-  top: ${props => props.languageSelectorHeight + dimensions.headerHeightLarge}px;
-  margin-top: ${props => props.languageSelectorHeight - dimensions.navigationMenuHeight}px;
+const Aside = styled.aside`
+  top: ${dimensions.headerHeightLarge + topOffset}px;
```

Comment:
Why did you choose to remove/change those instead of just adding an offset on top of the previous set `top`/`margin-top`? This way, on some widths it breaks, see the following screenshot where the toolbar pokes through the language selector.
Imo this should be reverted and just an additional offset added to the top.
<img src="https://github.com/digitalfabrik/integreat-app/assets/29095487/56466e66-c286-4264-958d-d325bf36dafe" width="400px" />

Can be reproduced here with a width e.g. 782px:
http://localhost:9000/testumgebung/de/events

Reply:
I don't think the toolbar needs to be fully visible when the language selectors are open, just like the top of the rest of the page isn't visible then. Though it should also not poke through, of course.

Reply:
We tried to make it not poke through before a few times and we did not manage because of various z-indices set across the app. So the workaround is to just always make it fully visible (below the language selector). However, if you got an idea on how to fix it that would be amazing of course! But otherwise, this should be fixed by adding enough top margin/top offset.

Reply:
Aaaaaah, I see, thank you for the explanation :) Looks like the `aside` having a new stacking context from being sticky is the culprit. I'll see if I find a solution for that, or do you have one, @relentless95 ?

Reply:
I agree also agree with you both. I just thought that it didn't matter whether the toolbar is visible when trying to access the  different languages because the goal is to access change the language. But i can just revert it, just let me know your final decision.

Reply:
It can be fixed by adding a `z-index: 0` to the `Body` (i.e. the parent of the `Aside`) in the `Layout.tsx`. That's because each stacking context is treated like one single element by the parent container when it comes to text flow. So we need to change the stacking order in that parent container (in this case the `Body`) to affect the `z-index` of the stacking context.

Reply:
Ah, I see, that puts the feedback module behind the header because it's a child of the aside. Too bad. I think fixing that is outside the scope of this ticket, so don't worry about it.
Which means that if we don't want the toolbar poking through, we need to put it lower than the language selectors. Unless you see another solution to not having both of them overlap there?

Reply:
@relentless95 are you still working on this? At least in my opinion most of the changes should be reverted and only this line should be changed in the following way:
``` ts
  top: ${props => props.languageSelectorHeight + dimensions.headerHeightLarge + additionalToolbarTopSpacing}px;
```
where `additonalToolbarTopSpacing` is something like `8`, `16` or `32` px or another order of 2. I think I like `32` but thats your choice :)

Reply:
Ah, didn't know that :see_no_evil: Lets see, I think we can get this ready :)